Submodule components/esp-azure/azure-iot-sdk-c contains modified content
Submodule c-utility contains modified content
diff --git a/components/esp-azure/azure-iot-sdk-c/c-utility/src/hmac.c b/components/esp-azure/azure-iot-sdk-c/c-utility/src/hmac.c
index 53f2411f..160af9d2 100644
--- a/components/esp-azure/azure-iot-sdk-c/c-utility/src/hmac.c
+++ b/components/esp-azure/azure-iot-sdk-c/c-utility/src/hmac.c
@@ -208,7 +208,7 @@ int hmacFinalBits(HMACContext *ctx,
 *   sha Error Code.
 *
 */
-int hmacResult(HMACContext *ctx, uint8_t *digest)
+int hmacResult(HMACContext *ctx, uint8_t digest[USHAMaxHashSize])
 {
     if (!ctx) return shaNull;
 

